examples/basic/shared/src/main/scala/sample/algebra/DocumentedApi.scala

```diff
@@ -0,0 +1,40 @@
+package sample.algebra
+
+import io.circe.generic.JsonCodec
+
+trait DocumentedApi
+  extends endpoints.algebra.DocumentedEndpoints
+    with endpoints.algebra.DocumentedOptionalResponses
+    with endpoints.algebra.DocumentedBasicAuthentication
+    with endpoints.algebra.DocumentedJsonEntities {
```

Comment:
Have you considered `endpoints.algebra.docs.Endpoints` instead of `DocumentedEndpoints`. I'm not sure which is better, just wondering.